CI says:  
```
/__w/AdaptiveCpp/AdaptiveCpp/src/compiler/cbs/IRUtils.cpp:20:10: fatal error: 'llvm/IR/DebugProgramInstruction.h' file not found
```
Do we need an LLVM version ifdef around this include?

--- a/src/compiler/cbs/IRUtils.cpp
+++ b/src/compiler/cbs/IRUtils.cpp
@@ -17,7 +17,6 @@
 #include <llvm/Analysis/RegionInfo.h>
 #include <llvm/IR/DIBuilder.h>
 #include <llvm/IR/DebugInfo.h>
-#include <llvm/IR/DebugProgramInstruction.h>
 #include <llvm/IR/Dominators.h>
 #include <llvm/IR/IRBuilder.h>
 #include <llvm/IR/IntrinsicInst.h>

